tests/functional/test_change_email_address.py

```diff
@@ -0,0 +1,36 @@
+import pytest
+from flask import url_for
+
+q_email = 'some.one@digital.cabinet-office.gov.uk'
+
+email_depts = [('some.one@cabinetoffice.gov.uk',
+                'Cabinet Office')]
```

Comment:
Do not indent to an arbitrary column. Instead, break the line after the open bracket and indent one level. Eg:
```python
email_depts = [
    ('some.one@cabinet....', 'Cabinet Office')
]
```